build.gradle

```diff
@@ -67,6 +67,9 @@ subprojects {
 
         // Logs
         compileOnly'org.slf4j:slf4j-api'
+
+        //SQL named parameters binding
+        implementation group: 'org.httprpc', name: 'httprpc', version: '5.9'
```

Comment:
This seems to be a full fledge HTTP server and client framework, this is a lot to import just to have named parameter injection.
Moreover, it seems to be a one-man project so I'm not fan to include it.
It also seems to moved to https://github.com/HTTP-RPC/Kilo